app/src/main/res/values/strings.xml

```diff
@@ -146,7 +146,11 @@
     <string name="account_list_manage_battery_saver">Manage battery saver</string>
     <string name="account_list_low_storage">Storage space low. Android will not sync local changes immediately, but during the next regular sync.</string>
     <string name="account_list_manage_storage">Manage storage</string>
+    <string name="account_list_calendar_storage_disabled_title">Calendar provider missing</string>
```

Comment:
Technically, these strings are not used by the account list. So when the account list code is changed by some, they may remove these strings accidentally.

I didn't already thing where they belong to. Probably all together (including the ones from the account_list) into some common extra block.

By the way, we could only use the title/description strings and make two paragraphs in `AccountsScreen`. They we don't have to duplicate the strings.

Reply:
I will create a new block called "Sync warnings" then :+1: